src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs

```diff
@@ -29,7 +29,7 @@ public void CustomizeView(DummyNode model, Dynamo.Controls.NodeView nodeView)
             RenderOptions.SetBitmapScalingMode(dummyNodeImage, BitmapScalingMode.HighQuality);
 
             nodeView.inputGrid.Children.Add(dummyNodeImage);
-            model.Warning(model.GetDescription());
+            model.Warning(model.GetDescription(), true);
```

Comment:
Make dummy node (unresolved node) warnings persistent over here to treat this type consistently with other types of persistent warnings.